libs/store/src/lib/infrastructure/persistence/store/entity-store.ts

```diff
@@ -70,7 +79,7 @@ export class DefaultEntityStore<T extends BaseEntity> implements EntityStore<T>
     }
```

Comment:
When getting an entity from the EntityService here:
```ts
    get(id: IdentityKey): Observable<T> {
        return this._entityService.getByKey(id);
    }
```

Does the EntityService return an Observable with the Entity or EntityDTO?

Shouldn't we be calling `instanceForType` here? For example:

```ts
    get(id: IdentityKey): Observable<T> {
        return this._entityService.getByKey(id).pipe(
            map(dto: EntityDTO => instanceForType(this._entity, dto))
        );
    }
```

Reply:
I think we should be returning Entity, EntityDTO is an internal thing